index.js

```diff
@@ -65,7 +80,7 @@ function copy(text, options) {
     debug && console.error("unable to copy using execCommand: ", err);
     debug && console.warn("trying IE specific stuff");
     try {
-      window.clipboardData.setData(options.format || "text", text);
+      window.clipboardData.setData("text", text);
```

Comment:
I checked the documentation on [microsoft website](https://msdn.microsoft.com/en-us/ie/ms535220(v=vs.80)) and there's no possibility to pass rich text. So I reverted to the previous code here.